app/models/alchemy/content.rb

```diff
@@ -177,6 +177,10 @@ def has_validations?
       definition['validate'].present?
     end
 
+    def classes
+      definition['classes'] || ''
```

Comment:
We already have the `settings` Hash. We do not need to introduce a new attribute on the model. Also we have a presenter class (`ContentEditor`) that is a perfect candidate for this.